app/controllers/api/v0/shipments_controller.rb

```diff
@@ -113,10 +115,9 @@ def get_or_create_shipment(stock_location_id)
       end
 
       def shipment_params
-        params.permit(
-          [:id, :order_id, :variant_id, :quantity,
-           { shipment: [:tracking, :selected_shipping_rate_id] }]
-        )
+        return {} unless params.has_key? :shipment
```

Comment:
Why do we want to return {} instead of letting ActionController::Params do its thing and raise an error if it's not there?